scripts/audit-governance-settings.sh

```
@@ -0,0 +1,488 @@
+  "$EXPECTED_BRANCH_FILE" \
+  "Branch ruleset enforcement must stay active."
+
+EXPECTED_TAG_ENFORCEMENT="$(jq -r '.enforcement' <<<"$EXPECTED_TAG_RULESET")"
+LIVE_TAG_ENFORCEMENT="$(jq -r '.enforcement // "missing"' <<<"$LIVE_TAG_RULESET")"
+record_comparison \
+  "tag-ruleset-active" "tag_protection" "high" \
+  "$EXPECTED_TAG_ENFORCEMENT" "$LIVE_TAG_ENFORCEMENT" \
+  "$EXPECTED_TAG_FILE" \
+  "Tag ruleset enforcement must stay active."
+
+EXPECTED_BRANCH_RULES="$(jq -c "$NORMALIZE_JQ
```

Shellcheck (0.11.0)

[warning] Did you forget to close this double quoted string?

(SC1078)